tests/pytests/test_kaguya_drivers.py

```diff
@@ -1,119 +1,185 @@
+                                   [  193267.90484827,   209421.16305764, -1767500.49120063],
+                                   [  192711.44397634,   208780.32318594, -1767633.01318365]]),
+            'velocities': np.array([[-1069.71225785, -1231.97438089,  -258.37880523],
+                                    [-1069.80205939, -1232.06556741,  -257.5939851 ],
+                                    [-1069.89161639, -1232.15647191,  -256.80910187],
+                                    [-1069.98092881, -1232.24709434,  -256.02415587],
+                                    [-1070.06999666, -1232.33743471,  -255.2391471 ],
+                                    [-1070.15881991, -1232.42749298,  -254.4540759 ]]),
+             'unit': 'm'},
+         'sun_position': {
+            'positions': np.array([[9.50347435e+10, 1.15913462e+11, 3.78747288e+09]]),
+            'velocities': np.array([[ 285730.03370329, -232701.52529823, 592.79276721]]),
+            'unit': 'm'},
+         'sensor_orientation': {
+            'quaternions': np.array([[-0.19095485, -0.08452708,  0.88748467, -0.41080698],
+                                     [-0.19073945, -0.08442789,  0.88753312, -0.41082276],
+                                     [-0.19052404, -0.08432871,  0.88758153, -0.41083852],
+                                     [-0.19030862, -0.08422952,  0.88762988, -0.41085426],
+                                     [-0.19009352, -0.08412972,  0.88767854, -0.41086914],
+                                     [-0.18987892, -0.08402899,  0.88772773, -0.41088271]])},
+        'detector_sample_summing': 1,
+        'detector_line_summing': 1,
+        'focal_length_model': {
+            'focal_length': 72.45},
+        'detector_center': {
+            'line': 0.5,
+            'sample': 2048.0},
+        'starting_detector_line': 0,
+        'starting_detector_sample': 1,
+        'focal2pixel_lines': [0, -142.85714285714286, 0],
+        'focal2pixel_samples': [0, 0, -142.85714285714286],
+        'optical_distortion': {
+            'kaguyatc': {
+                'x': [-0.0009649900000000001, 0.00098441, 8.5773e-06, -3.7438e-06],
+                'y': [-0.0013796, 1.3502e-05, 2.7251e-06, -6.193800000000001e-06]}},
+        'image_lines': 400,
+        'image_samples': 3208,
+        'name_platform': 'SELENE-M',
+        'name_sensor': 'Terrain Camera 1',
+        'reference_height': {
+            'maxheight': 1000,
+            'minheight': -1000,
+            'unit': 'm'},
+        'name_model': 'USGS_ASTRO_LINE_SCANNER_SENSOR_MODEL',
+        'interpolation_method': 'lagrange',
+        'line_scan_rate': [[0.5, -1.300000011920929, 0.006500000000000001]],
+        'starting_ephemeris_time': 292234259.82293594,
+        'center_ephemeris_time': 292234261.12293595,
+        't0_ephemeris': -1.300000011920929,
+        'dt_ephemeris': 0.5200000047683716,
+        't0_quaternion': -1.300000011920929,
+        'dt_quaternion': 0.5200000047683716}
```

Comment:
This looks like a pds3 usgscsm isd, I thought we wanted to compare against an ISIS usgscsm isd?

Reply:
We want to test against both. I can co back in and add the ISIS comparison in another PR. I want to get the conftest.py stuff in here so that other PRs can use it.